standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/IMetaStoreClient.java

```diff
@@ -2893,6 +2880,16 @@ void replCommitTxn(long txnid, String replPolicy)
    */
   List<TxnToWriteId> allocateTableWriteIdsBatch(List<Long> txnIds, String dbName, String tableName) throws TException;
 
+  /**
+   * Allocate a per table write ID and associate it with the given transaction. Used by replication load task.
+   * @param dbName name of DB in which the table belongs.
+   * @param tableName table to which the write ID to be allocated
+   * @param replPolicy Used by replication task to identify the source cluster.
+   * @param txnToWriteIdList List of txn to write id map.
```

Comment:
Mention clearly that the txn id in this map is source txn id. Better to rename the argument as srcTxnToWriteIdList

Reply:
done